nixos/modules/services/web-apps/hotcrp.nix

```
@@ -0,0 +1,199 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  inherit (lib) mkDefault mkEnableOption mkForce mkIf mkMerge mkOption types;
+  inherit (lib) concatStringsSep literalExpression mapAttrsToList mapAttrs' nameValuePair filterAttrs mapAttrs optional optionalString;
+  dataDirForInstance = instanceName: "/var/www/${cfg.user}/${instanceName}"; # compute data directory for an instance.
+  cfg = config.services.hotcrp;
+
+  phpExt = pkgs.php81.buildEnv {
+    extensions = { all, ... }: with all; [ mbstring intl mysqlnd ];
+  };
+in
+  {
+  # Interface
+  options.services.hotcrp = {
+    enable = mkEnableOption (lib.mdDoc "Hotcrp web application");
```

```suggestion
    enable = mkEnableOption "Hotcrp web application";
```
`lib.mdDoc` is now just an alias and can be safely removed everywhere.
see https://github.com/NixOS/nixpkgs/commit/d36f950d405369abca5144b14a201bc25c4a080a and https://github.com/NixOS/nixpkgs/pull/237557